src/pdf.ts

```diff
@@ -68,44 +109,115 @@ export async function fillPdf(
+    setFieldText(startMonthField, `${params.start.getMonth() + 1}`.padStart(2, '0'), font);
+    setFieldText(startYearField, `${params.start.getFullYear()}`, font);
+    setFieldText(endMonthField, `${params.end.getMonth() + 1}`.padStart(2, '0'), font);
+    setFieldText(endYearField, `${params.end.getFullYear()}`, font);
 
-    tableATax035QuantityField.setText(`${params.tableATax035Quantity}`);
-    tableATax035TaxBaseField.setText(`${formatMoney(params.tableATax035TaxBase)}`);
-    tableATax035TaxValueField.setText(`${formatMoney(params.tableATax035TaxValue)}`);
+    setFieldText(nationalRegistrationNumberField, params.nationalRegistrationNumber, font);
 
-    tableATax132QuantityField.setText(`${params.tableATax132Quantity}`);
-    tableATax132TaxBaseField.setText(`${formatMoney(params.tableATax132TaxBase)}`);
-    tableATax132TaxValueField.setText(`${formatMoney(params.tableATax132TaxValue)}`);
+    setFieldText(fullNameField, params.fullName, font);
 
-    tableATotalTaxValue.setText(`${formatMoney(params.tableATotalTaxValue)}`);
-    totalTaxValue.setText(`${formatMoney(params.totalTaxValue)}`)
-    locationField.setText(params.location);
-    dateField.setText(params.date);
+    setFieldText(addressLine1Field, params.addressLine1, font);
+    setFieldText(addressLine2Field, params.addressLine2, font);
+    setFieldText(addressLine3Field, params.addressLine3, font);
+
+    setFieldText(tableATax012QuantityField, `${params.tableATax012Quantity}`, font);
+    setFieldText(tableATax012TaxBaseField, formatMoney(params.tableATax012TaxBase, moneyCur, moneySep), font);
+    setFieldText(tableATax012TaxValueField, formatMoney(params.tableATax012TaxValue, moneyCur, moneySep), font);
+
+    setFieldText(tableATax035QuantityField, `${params.tableATax035Quantity}`, font);
+    setFieldText(tableATax035TaxBaseField, formatMoney(params.tableATax035TaxBase, moneyCur, moneySep), font);
+    setFieldText(tableATax035TaxValueField, formatMoney(params.tableATax035TaxValue, moneyCur, moneySep), font);
+
+    setFieldText(tableATax132QuantityField, `${params.tableATax132Quantity}`, font);
+    setFieldText(tableATax132TaxBaseField, formatMoney(params.tableATax132TaxBase, moneyCur, moneySep), font);
+    setFieldText(tableATax132TaxValueField, formatMoney(params.tableATax132TaxValue, moneyCur, moneySep), font);
+
+    setFieldText(tableATotalTaxValue, formatMoney(params.tableATotalTaxValue, moneyCur, moneySep), font);
+    setFieldText(totalTaxValue, formatMoney(params.totalTaxValue, moneyCur, moneySep), font);
+    setFieldText(locationField, params.location, font);
+    setFieldText(dateField, params.date, font);
 
     if(params.signaturePng) {
         signatureField.setImage(await document.embedPng(params.signaturePng));
     }
-    signatureNameField.setText(params.signatureName);
-    signatureCapacityField.setText(params.signatureCapacity);
+    setFieldText(signerField, `${params.signatureName}  , ${params.signatureCapacity}`, font);
```

Comment:
Probably best to omit the comma if no capacity is provided.
```suggestion
    let signer = params.signatureName;
    if (params.signatureCapacity.length > 0) {
        signer += `  , ${params.signatureCapacity}`;
    }
    setFieldText(signerField, signer, font);
```